e2e/src/usecases/ChooseYourAdventure.js

```diff
@@ -0,0 +1,56 @@
+import { launchApp } from '../utils/retries'
+import { quickOnboarding, sleep, waitForElementId } from '../utils/utils'
+
+export default ChooseYourAdventure = () => {
+  beforeEach(async () => {
+    await device.uninstallApp()
+    await device.installApp()
+    await launchApp({
+      launchArgs: {
+        statsigGateOverrides: `use_tab_navigator=true,show_cloud_account_backup_setup=true,show_cloud_account_backup_restore=true`,
+      },
+    })
+    await quickOnboarding({ stopOnCYA: true, cloudBackupEnabled: true })
+  })
+
+  it('learn about celo navigates to celo token details page', async () => {
+    await element(by.text('Learn about Celo')).tap()
+
+    // Check that we are on the Celo token details page
+    await waitForElementId('TokenDetails/AssetValue')
+    await expect(element(by.text('Celo')).atIndex(0)).toBeVisible()
+
+    // Back should go to the home screen
+    await element(by.id('BackChevron')).tap()
```

Comment:
```suggestion
    await waitForElementByIdAndTap('BackChevron')
```

This the advantage of waiting for the back chevron before attempting to tap, but this should also be handled by detox's sync.

Reply:
We wait for other elements of the screen and have assertions, so back should be visible at this point.